ui/app/routes/vault/cluster/access/method/item/show.js

```diff
@@ -11,9 +11,8 @@ export default Route.extend({
     const methodModel = this.modelFor('vault.cluster.access.method');
     const { type } = methodModel;
     const modelType = `generated-${singularize(itemType)}-${type}`;
-    return this.store.findRecord(modelType, itemName, {
-      adapterOptions: { path: `${method}/${itemType}` },
-    });
+    const id = `${method}/${itemName}`;
```

Comment:
I'm surprised that passing the method path as part of the id doesn't break things - is this split in the serializer or the adapter somewhere?

Reply:
your hunch was spot on! handling ids differently on show and list pages added unnecessary complexity and broke the ability to delete an item from the show instead of list page.

this was fixed by using `queryRecord` and instead of `findRecord` and always passing in the `id` and `authMethodPath` (which is exactly what you were suggesting below. here's an example of that: https://github.com/hashicorp/vault/pull/7513/files#diff-bb0dce24d1fc19d814e2991e47d29f75R12

thank you for your help!